packages/core/core/src/requests/AssetGraphRequest.js

```diff
@@ -523,8 +523,10 @@ export class AssetGraphBuilder {
           if (assetGroups.length === 1) {
             let [assetGroupId] = assetGroups;
             let assetGroup = nullthrows(this.assetGraph.getNode(assetGroupId));
-            invariant(assetGroup.type === 'asset_group');
```

Comment:
It's possible for a dependency to point directly to an asset, without an asset group. This happens when an asset references another asset returned by the same transformer by uniqueKey. See `resolveAsset` in AssetGraph.js.

Reply:
Did you hit this assertion? Should this actually be handled somehow and not just ignored?

Reply:
I did hit it due to our weird transformer that returns multiple assets and a dependency between them. I wasn't exactly sure how to handle it. What do you think?

Reply:
Yes, it should take these dependencies into account as well. I'll create a PR for that